lib/api/api-fetch/headers.js

```diff
@@ -49,21 +53,21 @@ function isHeaders (object) {
 function fill (headers, object) {
   if (isHeaders(object)) {
     // Object is instance of Headers
-    headers[kHeadersList] = Array.splice(object[kHeadersList])
+    headers[kHeadersList] = [...object[kHeadersList]]
```

Comment:
@Ethan-Arrowood

Reply:
It actually failed the test with `Array.splice` :/

Reply:
Ohhh this is an error in my code. It should be `object[kHeadersList].splice(0)`

Reply:
(but also spread operator is fine too)

Reply:
https://github.com/nodejs/node/pull/38986/files?file-filters%5B%5D=.js&file-filters%5B%5D=.mjs#diff-3569a54af4e25669eee4cc77bd0f4e86ca5fafb0cf766759a4bd2fb482fa0984R95